ftfy/badness.py

```diff
@@ -35,9 +35,6 @@ def _make_weirdness_regex():
     """
     groups = []
 
-    # Match lowercase letters that are followed by non-ASCII uppercase letters
-    groups.append('lA')
```

Comment:
Just out of curiosity, why this change?

Reply:
It's not even doing what it says, and is redundant with `[Ll][AaC]` a few lines down.

Reply:
But yes, good that you asked, as I didn't describe this change.

Reply:
Gosh, it sure is.  (Is redundant with below, I mean.)  Cool beans.